### ISSUE TITLE
Is this repo up-to-date?

### ISSUE THREAD

**Issue by Flimm** [atlemo/SubtlePatterns#25]
The website seems to contain more patterns that this repo.

**Comment by Joeao** [comment]
Yeah it's not up to date. I updated it myself a few years back but never made a pull request and have lost the changes. I regret.

**Comment by Joeao** [comment]
Made a new pull request: https://github.com/atlemo/SubtlePatterns/pull/26

**Comment by digideth** [comment]
Atle sold it to Toptal in 2017 according to his website http://atle.co/portfolio-item/subtle-patterns/

It was a great and lively website we he ran it but since then it has pretty much died.

Sad sad sad.
